assets/app/views/_templateopt.html

```diff
@@ -0,0 +1,21 @@
+<div class="template-options" ng-show="hasParameters">
+  <div ng-show="optionsExpanded">
+    <h3>Template options</h3> 
+    <a href ng-click="toggleOptionsExpanded()">Collapse</a> 
+    <ul class="options" ng-repeat="parameter in template.parameters">
+      <li>
+        <label title="{{ parameter.description }}">{{parameter.name}}</label>
```

Comment:
if we are going to set the description as the title lets set the cusor style to help and make it a data-toggle=tooltip